DataAccessLayer/Realization/HeroService.cs

```diff
@@ -0,0 +1,59 @@
+﻿using DataAccessLayer.Interfaces;
+using DataAccessLayer.Core;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Data.Entity;
+
+namespace DataAccessLayer.Realization
+{
+    public class HeroService : IRepository<Heroes>
```

Comment:
Naming:
Services implement business logic. They don't have to implement data access stuff.
Please, rename this to be HeroRepository.